tests/integration/files/conf/cloud.profiles.d/linode.conf

```diff
@@ -1,11 +1,5 @@
 linode-test:
   provider: linode-config
-  size: Linode 1GB
-  image: Alpine 3.12
-  location: Newark, NJ, USA
-
-linode-test-v4:
-  provider: linode-config-v4
   size: g6-standard-1
   image: linode/alpine3.12
```

Comment:
This might cause issue becuase `linode/alpine3.12` is no longer available on Linode.com

Let me update it very quick